src/config/configAggregator.ts

```diff
@@ -170,11 +170,16 @@ export class ConfigAggregator extends AsyncOptionalCreatable<JsonMap> {
    * @param key The key of the property.
    */
   public getPropertyValue<T extends AnyJson>(key: string): Optional<T> {
-    if (this.getAllowedProperties().some((element) => key === element.key)) {
-      return this.getConfig()[key] as T;
-    } else {
+    if (!this.getAllowedProperties().some((element) => key === element.key)) {
```

Comment:
All these changes to ConfigAggregator shouldn't be needed once https://github.com/salesforcecli/plugin-config/pull/299 is merged

Reply:
How does a change to plugin-config mitigate this change?

Reply:
Sorry, wrong PR. This is the sfdx-core one: https://github.com/forcedotcom/sfdx-core/pull/563

Reply:
@mdonnalley Did you want us to wait for PR #563 to be merged before we merge this one?

Reply:
@peternhale Yes. I don't want to merge in any of the changes to ConfigAggregator that were made in this PR